hub/test/src/testDrivers.js

```diff
@@ -394,7 +390,41 @@ function testGcDriver() {
   })
 }
 
+function testInMemoryDriver() {
+  test('InMemory Driver', async (t) => {
+    const driver = await InMemoryDriver.spawn()
+    try {
+      const fetch = require('node-fetch')
+
+      const prefix = driver.getReadURLPrefix()
+      const contentBuff = Buffer.from('hello world')
+      const s = new Readable()
+      s.push(contentBuff)
+      s.push(null)
+
+      const readUrl = await driver.performWrite(
+          { path: 'foo.txt',
+            storageTopLevel: '12345',
+            stream: s,
+            contentType: 'application/octet-stream',
+            contentLength: contentBuff.length })
+
+      t.ok(readUrl.startsWith(prefix + '12345'), `${readUrl} must start with readUrlPrefix ${prefix}12345`)
+      const resp = await fetch(readUrl)
+      const resptxt = await resp.text()
+      t.equal(resptxt, 'hello world', `Must get back hello world: got back: ${resptxt}`)
```

Comment:
This test also needs to verify that the content type matches what was given on the call to `performWrite`.